src/recap

```diff
@@ -233,28 +233,28 @@ print_sar() {
   sar "${FLAGS}" >> "${LOGFILE}"
 }
 
-# print the output of "apachectl fullstatus" to the specified file
-print_httpd_fullstatus() {
+# Print the output of the web server status page to the specified file
+print_http_fullstatus() {
   local LOGFILE="$1"
-  echo "Apache Status report" >> "${LOGFILE}"
-  apachectl fullstatus >> "${LOGFILE}"
+  echo "Web Status report" >> "${LOGFILE}"
+  curl -Ls "${OPTS_STATUSURL}" 2>/dev/null >> "${LOGFILE}"
```

Comment:
Indeed. There are more use cases where different options can be useful, a quick one is `-v` so I've added that.